src/stack/phy/layer/LtePhyRelay.cc

```diff
@@ -27,7 +29,7 @@ void LtePhyRelay::initialize(int stage)
         WATCH(nodeType_);
         WATCH(masterId_);
     }
-    else if (stage == 1)
+    else if (stage == inet::INITSTAGE_PHYSICAL_ENVIRONMENT)
```

Comment:
This stage is for the physical environment such as physical objects, ground model, etc. There's another one for the physical layer called INITSTAGE_PHYSICAL_LAYER.

Reply:
Was changed to INITSTAGE_PHYSICAL_LAYER.